expected/corner_cases_3.out

```diff
@@ -0,0 +1,67 @@
+Parsed test spec with 2 sessions
```

Comment:
Can you, please, add info into review section of README (https://github.com/postgrespro/pg_query_state#tests) about Postgres/OS versions where we need this new variants of test outputs (corner_cases_3.out, corner_cases_4.out)?

Reply:
Yes, I updated README.md and added this information (in commit a586969).